uberfire-project/uberfire-project-api/src/main/java/org/guvnor/common/services/project/service/WorkspaceProjectService.java

```diff
@@ -44,6 +44,9 @@ WorkspaceProject newProject(final OrganizationalUnit organizationalUnit,
                                 final POM pom,
                                 final DeploymentMode mode);
 
+    String createProjectName(OrganizationalUnit organizationalUnit,
+                             String name);
```

Comment:
Could we please use `final OrganizationalUnit organizationalUnit` and `final String name`?